src/sentry/mediators/project_rules/creator.py

```diff
@@ -1,21 +1,21 @@
-from collections.abc import Iterable
+from rest_framework.request import Request
 
 from sentry.mediators.mediator import Mediator
 from sentry.mediators.param import Param
-from sentry.models import Actor, Rule
+from sentry.models import Actor, Project, Rule
 
 
 class Creator(Mediator):
-    name = Param((str,))
+    name = Param(str)
     environment = Param(int, required=False)
-    owner = Param("sentry.models.Actor", required=False)
-    project = Param("sentry.models.Project")
-    action_match = Param((str,))
-    filter_match = Param((str,), required=False)
-    actions = Param(Iterable)
-    conditions = Param(Iterable)
```

Comment:
this is technically a functional change -- mypy does not allow non-concrete classes for type parameters

Reply:
ah the github diff comment doesn't show the new code -- basically did `Iterable` => `list`